test/test_helper.rb

```diff
@@ -18,6 +18,7 @@
 $VERBOSE = nil unless ENV["VERBOSE"] || ENV["CI"]
 
 require_relative "../lib/ruby_lsp/internal"
+require_relative "../lib/ruby_lsp/test_helper"
```

Comment:
I know it's not related to this PR, but can we please update all of these to use regular requires? The `lib` directory gets inserted into the `$LOAD_PATH` a few lines above, there's no need to use relative requires.